### PR TITLE
modify staccato to use google analytics forward proxy

### PR DESCRIPTION
**Pull request by lihanli**
## Description of change
modify staccato library to send requests through the google analytics forward proxy instead of directly to google analytics

## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/13856

## Things to know about this PR

tested in staging

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -572,6 +572,9 @@ github_stats:
   username: github-stats-rake
   token: fake_token
 
+google_analytics:
```
Comment (erluti) [inline config/settings.yml:575]: settings.yml has a reference to `google_analytics_tracking_id` on line 457.  That should probably move in with this group, it's only used in a few places.

Comment (erluti) [reply config/settings.yml:575]: just occured to me that would require a corresponding change to devops.
hopefully the keys could be side-by-side if changing devops is too complicated

Comment (lihanli) [reply config/settings.yml:575]: thanks for the suggestion i made a new issue for it https://github.com/department-of-veterans-affairs/va.gov-team/issues/14017

**Review by erluti — APPROVED** [review]
thanks for creating that ticket

**Pull request merged**